shtab does not generating working completions

**Describe the bug**
When trying to complete file paths for trash to operate on, nothing is completed. In zsh, the description of the thing that is trying to complete is "trash commands". In bash, nothing is completed, and no messages are displayed. Options seem to work in both bash and zsh.

**trash-cli version**
Output of: trash-put --version
`0.22.10.20`

**Are you using the latest version of trash-cli?**
Yes

**Have you tried if the bug is present in the latest version of trash-cli?**
Yes (arch's system version and git version in a python venv)

**Operating system:**
 - OS: arch

**To Reproduce**
Copy and paste the commands (and their output) to execute in order to reproduce 
the behavior:

1. `trash-put <tab>`

**Expected behavior**
The completion for the shell should include paths visible to the user.

Thanks for the quick fix, but the tests now fail as per below. Reverting this commit allows the tests to pass again.

```
================================================================ FAILURES =================================================================
______________________________________________ TestPutScripts.test_trashes_connected_symlink ______________________________________________

self = <tests.test_put.test_trash_put_script.TestPutScripts testMethod=test_trashes_connected_symlink>

    def test_trashes_connected_symlink(self):
        self.make_connected_link(self.tmp_dir / 'link')
    
        result = run_command(self.tmp_dir, 'trash-put', [
            'link',
            '--trash-dir', self.tmp_dir / 'trash-dir',
        ])
    
        assert (result.stdout, result.stderr == "", "")
        assert not os.path.lexists(self.tmp_dir / 'link')
>       assert os.path.lexists(self.tmp_dir / 'trash-dir' / 'files' / 'link')
E       AssertionError: assert False
E        +  where False = <function lexists at 0x7fa83f0929e0>(((('/tmp/tmpe3hn4anw_trash_cli_test' / 'trash-dir') / 'files') / 'link'))
E        +    where <function lexists at 0x7fa83f0929e0> = <module 'posixpath' from '/usr/lib/python3.10/posixpath.py'>.lexists
E        +      where <module 'posixpath' from '/usr/lib/python3.10/posixpath.py'> = os.path
E        +    and   '/tmp/tmpe3hn4anw_trash_cli_test' = <tests.test_put.test_trash_put_script.TestPutScripts testMethod=test_trashes_connected_symlink>.tmp_dir

tests/test_put/test_trash_put_script.py:48: AssertionError
______________________________________________ TestPutScripts.test_trashes_dangling_symlink _______________________________________________

self = <tests.test_put.test_trash_put_script.TestPutScripts testMethod=test_trashes_dangling_symlink>

    def test_trashes_dangling_symlink(self):
        self.make_dangling_link(self.tmp_dir / 'link')
    
        result = run_command(self.tmp_dir, 'trash-put', [
            'link',
            '--trash-dir', self.tmp_dir / 'trash-dir',
        ])
    
        assert (result.stdout, result.stderr == "", "")
        assert not os.path.lexists(self.tmp_dir / 'link')
>       assert os.path.lexists(self.tmp_dir / 'trash-dir' / 'files' / 'link')
E       AssertionError: assert False
E        +  where False = <function lexists at 0x7fa83f0929e0>(((('/tmp/tmpl5mshxnm_trash_cli_test' / 'trash-dir') / 'files') / 'link'))
E        +    where <function lexists at 0x7fa83f0929e0> = <module 'posixpath' from '/usr/lib/python3.10/posixpath.py'>.lexists
E        +      where <module 'posixpath' from '/usr/lib/python3.10/posixpath.py'> = os.path
E        +    and   '/tmp/tmpl5mshxnm_trash_cli_test' = <tests.test_put.test_trash_put_script.TestPutScripts testMethod=test_trashes_dangling_symlink>.tmp_dir

tests/test_put/test_trash_put_script.py:36: AssertionError
============================================================ warnings summary =============================================================
tests/test_put/test_trash_put_script.py:46
  /HDD/lee/.SDDsymlinks/home/lee/.cache/yay/trash-cli-git/src/trash-cli/tests/test_put/test_trash_put_script.py:46: PytestAssertRewriteWarning: assertion is always true, perhaps remove parentheses?
    assert (result.stdout, result.stderr == "", "")

tests/test_put/test_trash_put_script.py:34
  /HDD/lee/.SDDsymlinks/home/lee/.cache/yay/trash-cli-git/src/trash-cli/tests/test_put/test_trash_put_script.py:34: PytestAssertRewriteWarning: assertion is always true, perhaps remove parentheses?
    assert (result.stdout, result.stderr == "", "")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
========================================================= short test summary info =========================================================
FAILED tests/test_put/test_trash_put_script.py::TestPutScripts::test_trashes_connected_symlink - AssertionError: assert False
FAILED tests/test_put/test_trash_put_script.py::TestPutScripts::test_trashes_dangling_symlink - AssertionError: assert False
================================================ 2 failed, 388 passed, 2 warnings in 3.26s ================================================
```

--- a/trashcli/put/parser.py
+++ b/trashcli/put/parser.py
@@ -105,6 +105,6 @@ Report bugs to https://github.com/andreafrancia/trash-cli/issues""")
                         action="version",
                         version=version)
     parser.add_argument('files',
-                        nargs='...'
+                        nargs='*'
                         ).complete = TRASH_FILES
     return parser

